lib/get-standings.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+/**
+  @param season Assumed to be the season format.
+  @param matches key-match object
+*/
+const getStandings = module.exports = function(season, matches) {
```

Comment:
Does it make sense to export a `function(season, matches)`, or should those be `require()`'d in? Would it make sense to just replace the logic in `model/seasons` with this method?

Reply:
I like the idea of having things more modular, and this is complex enough to warrant its own module. Not sure about imports, but for now it was the easiest way to plug it in where the old method lived.